config/psalm/psalm-baseline.xml

```diff
@@ -6365,11 +6353,6 @@
       <code>(int) $offset</code>
     </RedundantCast>
   </file>
-  <file src="includes/wpml/class-sensei-wpml.php">
```

Comment:
Is it here on purpose? Or did it come accidentally from another PR?

Reply:
It was removed automatically by Psalm. Probably didn't have that issue anymore.